src/iosMain/kotlin/org/cru/godtools/tool/IosParserConfig.kt

```diff
@@ -13,6 +13,13 @@ actual object ParserConfig {
         set(value) {
             _supportedDeviceTypes.value = if (value.isFrozen) value else value.toSet().freeze()
         }
+
+    private val _supportedFeatures = AtomicReference(emptySet<String>())
+    actual var supportedFeatures: Set<String>
+        get() = _supportedFeatures.value
+        set(value) {
+            _supportedFeatures.value = if (value.isFrozen) value else value.toSet().freeze()
```

Comment:
What is  ```isFrozen``` and ```freeze``` ?

Reply:
Kotlin/Native specific thing, basically data structures in Kotlin/Native are thread locked. The only way to access data on another thread is for that data to be `frozen` first. That makes the data immutables, so it can no longer be changed and introduce race conditions.

The frozen concept may be going away in the future once the new Kotlin/Native memory manager is released.

Kotlin/Native is what compiles this logic to run on iOS